src/pymor/vectorarrays/list.py

```diff
@@ -260,6 +260,10 @@ def make_array(cls, subtype=None, count=0, reserve=0):
         vector_type, vector_subtype = subtype
         return cls([vector_type.make_zeros(vector_subtype) for _ in xrange(count)], subtype=subtype, copy=False)
 
+    @classmethod
+    def from_data(cls, data, subtype):
+        return cls([NumpyVector(v) for v in data], subtype=subtype)
```

Comment:
I do not think that you always want to have `NumpyVectors`. `Vector` implementations from external solvers might implement `from_data` as well. Also with the current implementation, the resulting array should have subtype (NumpyVector, subtype). I would propose to add `from_data` to the `VectorInterface` as well and implement it here the same way as `make_array`.